webapp/publisher/snaps/views.py

```diff
@@ -215,6 +212,11 @@ def get_listing_snap(snap_name):
         m['url'] for m in snap_details['media']
         if m['type'] == 'screenshot']
 
+    try:
+        licenses = github_api.get_licenses()
```

Comment:
Do we really need to call API for licences every time we call for listing page? I guess that's not something that changes very often...

Reply:
this endpoint is super cached so we won't make that many api calls